### ISSUE TITLE
Issue 7 : Hyper-Parameter Tuning and Ensembles ( Lab 9 - 11)

### ISSUE THREAD

**Issue by TrevorA17** [TrevorA17/Loan_Default_Prediction#7]
- [x] Random/Grid/Manual Search
- [x] Bagging/Boosting/Stacking Ensembles (if applicable)

**Comment by TrevorA17** [comment]
most of the models have a similar accuracy of 0.83, it's possible that the default hyperparameters used by the models are already providing a good fit to the data.
